gizmo.py

```diff
@@ -0,0 +1,90 @@
+from pandas.core.frame import DataFrame
+
+
+def hello(name,country="Finland"):
+    print("Hello {0}, how are things in {1}?".format(str(name),str(country)))
+
+def spell(input_str):
+    result=input_str[0]
+    for i in input_str[1:]:
+        result+="."+i
+        
+    return print(result)
+
+def relative_path(subject_identifiers,relpath='./subjects/moc_recording'):
+    names=[]
+    relpath='./subjects/mock_recording'
+    for i in subject_identifiers:
+        res="{0}_{1}.rec".format(relpath,i) #new style
+        #res=relpath+"_"+i+".rec" #old style, not good
+        names.append(res)
+
+    return names
+
+class Gizmo:
+    def __init__(self, name):
+        self.name=name
+
+    def speak(self):
+        print(self.name)
+
+
+def multiplication_table(zero_out_multiples=None):
+    """ 
+    Calculates a multiplication table based on two vectors ranging from 1 to 12 each.
+
+    The function multiplication_table caclulates a multiplication table for all numbers between 1 to 12. If the input argument zero_out_multiples is not None, multiplies of this input argument in the table will be all set to zero.
+
+    Parameters
+    ----------
+    zero_out_multiples : int, optional
+        The arg is used for the multiplication table. If it is not None, the table returned by the function will have all multiples of the given number set to zero.
+
+    Returns
+    -------
+    res : numpy.ndarray
+        The arg contains the calculated multiplication table. Calculation is based on the outer product.
+    """
+    import numpy as np
+
+    a=np.arange(1,13,1)
+    b=np.arange(1,13,1)
+
+    if zero_out_multiples is not None and isinstance(zero_out_multiples, int):
```

Comment:
When `zero_out_multiples` is not an `int` value (but it is set, i.e. it is not `None`) you probably want to `raise` an error instead of silently doing nothing.

Reply:
it's important for users to get feedback if they are doing something wrong